### PR TITLE
VACMS-14729 PACT Act Wizard: adding Results page 1

### PR DESCRIPTION
**Pull request by randimays**
Adding the PACT Act wizard application basics and the first Results page (pages 1-2).

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14729

## Testing done
1. Navigate to `/test/pact-act`
2. Validate text against Danielle's copy for results page 1 ([Danielle's copy](https://dvagov-my.sharepoint.com/:w:/g/personal/cynthia_merrill_va_gov/EXcXBeLPnthDhzKkq0tHOgkB1hz4H5z-TpljngwAkGniFw?e=WJhfZI) - Requires CAG) and/or against Jordan's [design](https://www.sketch.com/s/2bf4405b-5cc4-4672-88fb-d215abdf3b0d)
3. Validate `Learn what to do next` link at the bottom of the page links to page 2
4. Validate text against copy or Jordan's design for page 2
5. Ensure that deep-linking redirects appropriately. I.e. from either results page (`/results-1-1` or `/results-1-2`), you get redirected to `/test/pact-act/results-1-1` when reloading the page.

#### Results set 1, page 1
Screenshot 2023-08-15 at 11 06 51 AM

#### Results set 1, page 2
Screenshot 2023-08-15 at 11 07 00 AM

##### Accordion 1
Screenshot 2023-08-15 at 11 07 11 AM

##### Accordion 2
Screenshot 2023-08-15 at 11 07 19 AM

#### Results set 1, page 1
Screenshot 2023-08-15 at 11 10 52 AM
<br/>
Screenshot 2023-08-15 at 11 10 57 AM

#### Results set 1, page 2
Screenshot 2023-08-15 at 11 11 09 AM
<br/>
Screenshot 2023-08-15 at 11 11 19 AM
<br/>
Screenshot 2023-08-15 at 11 11 30 AM
<br/>
Screenshot 2023-08-15 at 11 11 35 AM

## Acceptance criteria

- [x] Both screens of the R1 "results screen flow" are built, and linked by a link at the bottom of the first screen
- [x] The first screen is available at `/test/pact-act`
- [x] Confirmed that the screens conform to the design system (and are "working right") on both mobile and desktop devices

### CODE REVIEW

File: src/applications/pact-act/containers/results/1/Page1.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import { updateResultsPage1Viewed } from '../../../actions';
+import { customizeTitle } from '../../../utilities/customize-title';
+
+const ResultsSet1Page1 = ({
+  router,
+  updateResults1Viewed,
+  viewedResultsPage1,
+}) => {
+  const H1 = 'You may be eligible for VA benefits';
+  console.log('HELLO');
```
Comment (powellkerry) [inline src/applications/pact-act/containers/results/1/Page1.jsx:16]: This console statement probably should be removed

File: src/applications/pact-act/containers/results/1/Page2.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+import { waitForRenderThenFocus } from '@department-of-veterans-affairs/platform-utilities/ui';
+import { accordions } from '../../../constants/results-set-1-page-2-accordions';
+import { customizeTitle } from '../../../utilities/customize-title';
+import { ROUTES } from '../../../constants';
+
+const ResultsSet1Page2 = ({ viewedResultsPage1, router }) => {
+  const [hasScrolled, setHasScrolled] = useState(false);
+  const H1 = 'Apply for VA benefits now';
+
+  console.log('HELLO 2');
```
Comment (powellkerry) [inline src/applications/pact-act/containers/results/1/Page2.jsx:13]: This console statement should be removed

File: src/applications/pact-act/containers/results/1/Page1.jsx (hunk 1 above)
Comment (randimays) [reply src/applications/pact-act/containers/results/1/Page1.jsx]: Done 👍

**Comment by randimays** [comment]
@powellkerry Mind taking another look, please?

**Review by chriskim2311 — APPROVED** [review]
LGTM!, tested locally, verified all redirects and functionality of accordions, and content is identical to Danielle's

**Review by powellkerry — APPROVED** [review]

**Pull request merged**